crates/turborepo-scm/src/package_deps.rs

```diff
@@ -207,6 +242,45 @@ impl Git {
         hash_objects(&self.root, &full_pkg_path, to_hash, &mut hashes)?;
         Ok(hashes)
     }
+
+    #[tracing::instrument(skip(self, turbo_root, inputs))]
+    fn get_package_file_hashes_from_inputs_and_index<S: AsRef<str>>(
+        &self,
+        turbo_root: &AbsoluteSystemPath,
+        package_path: &AnchoredSystemPath,
+        inputs: &[S],
+        include_configs: bool,
```

Comment:
I don't think this param is needed
```suggestion
```

Reply:
Good catch! Left over from that function refactor